plugins/baseball-scoreboard/mlb_managers.py

```
@@ -0,0 +1,227 @@
+                    ESPN_MLB_SCOREBOARD_URL,
+                    params={"dates": datestring, "limit": 1000},
+                    headers=self.headers,
+                    timeout=30,
+                )
+                response.raise_for_status()
+                data = response.json()
+
+                # Cache the data
+                self.cache_manager.set(cache_key, data)
+                self.logger.info(f"Synchronously fetched {season_year} season schedule")
+                return data
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)